packages/trip-form/src/__mocks__/modes.js

```diff
@@ -0,0 +1,69 @@
+const commonModes = {
+  transitModes: [
+    {
+      mode: "BUS",
+      label: "Bus"
+    },
+    {
+      mode: "TRAM",
+      label: "MAX & Streetcar"
+    },
+    {
+      mode: "RAIL",
+      label: "WES"
+    },
+    {
+      mode: "GONDOLA",
+      label: "Aerial Tram"
+    }
+  ],
+  accessModes: [
+    // Biketown and Uber should be displayed the same despite Biketown company set in companies.
```

Comment:
This comment reads a bit cryptic, but seems important. Could you elaborate here?

Reply:
Yep. Fixed in lastest commit.